examples/generative/corrdiff/conf/model/corrdiff_diffusion_mini.yaml

```diff
@@ -18,6 +18,10 @@ defaults:
   - corrdiff_diffusion
 
 hr_mean_conditioning: True
+scale_cond_input: True
```

Comment:
For mini, this can be set to `False` as backward compatibility is not an issue.

Reply:
Changed to False